CHANGES.md

```diff
@@ -240,7 +240,14 @@ OpenSSL 3.2
 OpenSSL 3.1
 -----------
 
-### Changes between 3.0 and 3.1.0 [xx XXX xxxx]
+### Changes between 3.1.0 and 3.1.1 [xx XXX xxxx]
+
+ * Disallowed the use of truncated digests with Hash and HMAC DRBGs.
```

Comment:
This should mention that this affects only the FIPS provider and that there is an option to change the behavior.